src/components/EmptyMessage/EmptyMessage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const EmptyMessage = ({entityName, createCopy}) => (
+  <div>
+    <p>No {entityName} yet. <Link to={`${entityName}/create`}>{createCopy}</Link></p>
```

Comment:
Need to make the entityName lowercase in the link so the URL is exact 👍 

```<Link to={`${entityName.toLowerCase()}/create`}>```

Reply:
done